docs/INSTALLATION.md

```diff
@@ -0,0 +1,89 @@
+# Installation
+## Overview
+We understand you will probably be excited to power on your AmpliPro and try it out. To avoid damaging the unit, please read through this guide before installing and powering your AmpliPro unit!
+
+WARNING!! VERIFY THAT THE VOLTAGE OF YOUR WALL OUTLET MATCHES THE VOLTAGE LISTED NEAR THE POWER INLET BEFORE PLUGGING IN.
+
+See the power section for more details on how to change input voltage.
+
+## Mounting and unit placement
+
+Main Controllers and Zone Expanders both ship with built in 19" rackmount ears, so that they can be easily installed into a server rack with mounting screws (not provided).
+
+Some things to keep in mind avoid overheating the unit:
+- Don't block the vent holes on the side
+- Don't allow dust to build up inside the unit to avoid overheating the unit. See cleaning instructions page for more detail.
+
+
+## Speakers
+Each of AmpliPro's zone outputs can drive a 4-8 Ohm speaker pair, using the provided Phoenix connectors. Here is what the basic wiring diagram for a zone looks like:
```

Comment:
speaker pair sounds a bit odd - singular speaker implies a single speaker - which for a dual coil is correct. I would say something along the lines of each zone has two speaker outputs. I'm curious - did you get this from another manual? B/C if thats what they say for other amps we can adopt it.